pwnlib/commandline/hex.py

```diff
@@ -16,14 +17,38 @@
 parser.add_argument('data', nargs='*',
     help='Data to convert into hex')
 
+parser.add_argument(
+    '-p', '--prefix',
+    metavar = 'prefix',
+    type = str,
+    default = '',
+    help = 'Insert a prefix before each byte',
+)
+
+parser.add_argument(
+    '-s', '--separator',
+    metavar = 'separator',
+    type = str,
+    default = '',
+    help = 'Add a separator between each byte',
+)
+
+def format_hex(hex_string, prefix, separator):
+    return separator.join(["%s%s" % (prefix, x) for x in group(2, hex_string)])
```

Comment:
isn't it too complicated?
```suggestion
    return separator.join([prefix + x for x in group(2, hex_string)])
```